org.civicrm.reqangular/gulp-tasks/requirejs.js

```diff
@@ -0,0 +1,45 @@
+var exec = require('child_process').exec;
+var gulp = require('gulp');
+var path = require('path');
+var replace = require('gulp-replace');
+var templateCache = require('gulp-angular-templatecache');
+
+module.exports = function () {
+  var commonFolder = path.join(__dirname, '..', 'src/common');
+
+  return {
+    pre: [
+      {
+        name: 'cache-templates',
+        fn: function (cb) {
+          gulp.src(path.join(commonFolder, 'templates', '/**/*.html'))
+            .pipe(templateCache({
+              moduleSystem: 'RequireJS'
+            }))
+            .pipe(replace("['angular']", "['common/angular']"))
+            .pipe(replace('module(\'templates\')', 'module(\'common.templates\', [])'))
+            .pipe(gulp.dest(path.join(commonFolder, 'modules')));
+
+          cb();
+        }
+      },
+      {
+        name: 'requirejs:mock',
+        fn: function (cb) {
+          var buildMocksPath = path.join(__dirname, '..', 'build.mocks.js');
+
+          exec('r.js -o ' + buildMocksPath, function (err, stdout, stderr) {
+            err && err.code && console.log(stdout);
```

Comment:
Don't know to be honest, those are not simple messages but multi line messages with stack traces etc, i don't know how pleasant it would be to see this blob of red text

Reply:
Makes sense. Leave it white.